fetch.bs

```diff
@@ -2971,16 +2972,18 @@ boolean <var>http3Only</var>, run these steps:
   [[!HTTP1]] [[!TLS]]
 
   <ul>
-   <li><p>If <var>http3Only</var> is true, then establish an HTTP/3 connection. [[!HTTP3]]
+   <li>
+    <p>If <var>requireUnreliable</var> is true, then establish a connection capable of unreliable
+    transport, e.g., HTTP/3. [[!HTTP3]]
 
-   <li><p>When establishing an HTTP/3 connection, include SETTINGS_ENABLE_WEBTRANSPORT with a value
-   of 1 and H3_DATAGRAM with a value of 1 in the initial SETTINGS frame. [[!WEBTRANSPORT-HTTP3]]
-   [[!HTTP3-DATAGRAM]]
+    <p>When establishing such a connection, include SETTINGS_ENABLE_WEBTRANSPORT with a value of 1
+    and H3_DATAGRAM with a value of 1 in the initial SETTINGS frame. [[!WEBTRANSPORT-HTTP3]]
+    [[!HTTP3-DATAGRAM]]
```

Comment:
I would like some feedback on this part @ekinnear. Is this reasonable to desire for all unreliable transport connections? Presumably the details of this might shift in future protocol versions.

Perhaps this second paragraph should remain "When establishing an HTTP/3 connection"?

Reply:
+1, this seems like a good improvement, and as we have additional concrete options we can include the specific configuration to use for them.